package.json

```diff
@@ -47,8 +47,10 @@
     "html-loader": "0.4.3",
     "html-webpack-plugin": "2.22.0",
     "http-proxy-middleware": "0.17.1",
-    "jest": "22.4.4",
+    "intl-messageformat-parser": "1.4.0",
+    "jest": "24.0.0",
```

Comment:
This is the one that will require an update to nodejs-modules for an offline build to work.